process/throttle/antiflood/interface.go

```diff
@@ -0,0 +1,10 @@
+package antiflood
+
+// QuotaStatusHandler defines the behavior of a quota handler able to process periodic updates of peers quota measured
+// by the system
+type QuotaStatusHandler interface {
+	ResetStatistics()
+	AddQuota(identifier string, numReceivedMessages uint32, sizeReceivedMessages uint64,
```

Comment:
maybe use shorter names for parameters.

Reply:
Removed "Messages" part of all variable names.